src/lang/cs.js

```diff
@@ -0,0 +1,58 @@
+module.exports = function (number, index) {
+  let inflectionIndex = 0;
+
+  let isInflectionNeeded = index == 1 ||
+                           index == 3 ||
+                           index == 5 ||
+                           index == 7 ||
+                           index == 9 ||
+                           index == 11 ||
+                           index == 13;
+  if (isInflectionNeeded && number >= 5) {
+    inflectionIndex = 1;
+  }
+
+  return [
+    [['právě teď', 'právě teď']],
+    [
+      ['před %s vteřinami', 'za %s vteřiny'],
+      ['před %s vteřinami', 'za %s vteřin']
+    ],
+    [      ['před minutou', 'za minutu']    ],
+    [
+      ['před %s minutami', 'za %s minuty'],
+      ['před %s minutami', 'za %s minut']
+    ],
+    [      ['před hodinou', 'za hodinu']    ],
```

Comment:
please help me make the code style consistent. 😄

Reply:
Of course, what do you preffer? 

```
    [
      ['právě teď', 'právě teď']
    ],
    [
      ['před %s vteřinami', 'za %s vteřiny'],
      ['před %s vteřinami', 'za %s vteřin']
    ],
```

or
```
    [['právě teď', 'právě teď']],
    [
      ['před %s vteřinami', 'za %s vteřiny'],
      ['před %s vteřinami', 'za %s vteřin']
    ],
```

Reply:
all is ok, remove the redundant spaces will be ok.

Reply:
done